shared/profile/index.js

```diff
@@ -26,6 +39,19 @@ class Profile extends Component<void, Props, void> {
 export default connect(
   state => ({
     username: state.config.username,
+    trackers: state.tracker.trackers,
+  }),
+  dispatch => ({
+    refresh: username => dispatch(getProfile(username)),
   }),
-  dispatch => ({})
+  (stateProps, dispatchProps, ownProps) => {
+    // const username = ownProps.username || stateProps.username
+
+    return {
+      ...ownProps,
+      ...stateProps.trackers[username],
+      ...dispatchProps,
```

Comment:
the merge function should always do this, else if you add a dispatch it wont' show up which would be super confusing